p4-14/v1.0.3/tex/p4.tex

```diff
@@ -556,7 +556,7 @@
 maintain a separate valid indication which may be tested explicitly. Metadata 
 is always considered to be valid. This is further explained in 
 Section~\ref{sec:testing}.  Metadata instances are 
-initialized to 0 by default.
+initialized to 0 by default.  (TBD: Is this true?  Or are metadata instances uninitialized and contain undefined values initially?)
```

Comment:
bmv2 simple_switch initializes all metadata fields to 0 to be compatible with P4 programs and apps that were written with this assumption.